apps/docs/src/guide/testing/tweaking-the-blockchain.md

```diff
@@ -0,0 +1,23 @@
+# Producing Blocks
+
+You can force-produce blocks using the `produceBlocks` helper to achieve an arbitrary block height. This is especially useful when you want to do some testing regarding transaction maturity.
+
+<<< @/../../../packages/providers/src/provider.test.ts#Provider-produce-blocks{ts:line-numbers}
+
+# Producing Blocks With Custom Timestamps
+
+You can also produce blocks with custom timestamps using the `produceBlocks` helper by specifying the second optional parameter.
+
+```typescript
```

Comment:
Should this snippet be a referencing a unit test to ensure it does not get outdated? Looks like you've already got an applicable test for this also:

https://github.com/FuelLabs/fuels-ts/blob/73399468555ce70049f4c760ff77cd276a14ff5e/packages/providers/src/provider.test.ts#L212-L213

Reply:
Good catch @danielbate, looks like some unrelated changes crept in because of the base branch change. I updated the PR with `master`, and all seems ok now.